docs/DeveloperGuide.md

```diff
@@ -233,25 +233,35 @@ Step 5. After the project gets updated, `Model#saveProjectsFolder` is called to
 
 ### Add Event to Project Command [Implemented in v1.2]
```

Comment:
`[Implemented in v1.2]` can be removed since other sections do not have this.

Reply:
Thanks for pointing it out @lirc572 !